### DISCUSSION TITLE
Maccy version 2: what permissions are required?

### DISCUSSION THREAD

**Discussion by Kookaburra2027** [p0deje/Maccy#980] [Q&A] [answered]
Hello, can someone please tell me what permissions does Maccy (non MAS version) require on Sequioa (macOS 15.1.1)? 

After I installed Maccy for the first time, it worked without any permissions being granted so no Accessibility or Input permissions when I had read these might have been required. Just concerned I may have messed up the installation somehow or my system has issues.

Thanks in advance.

**Comment by p0deje** [comment, marked as answer]
The accessibility permissions are requested the first time you attempt to paste a copied item from Maccy (e.g. Option+click on item). If you don't use pasting, no extra permissions are needed.

**Reply by Kookaburra2027** [reply]
Thanks for your response. 

I was able to copy and paste without getting the Accessibility permission prompt. However, I did try to press option and click (as you mentioned) and then received the prompt. I declined but it still seems to allow me to recopy an item from Maccy history for pasting. 

Can you please let me know what type of pasting? I might not be understanding things correctly and apologies in advance.

**Reply by p0deje** [reply]
If you accept the prompt, Maccy would also paste directly to your focused app similar to you pressing Command+V.

**Reply by Kookaburra2027** [reply]
Thanks for explaining this to me. And good to know that Accessibility permissions are not required but for this use case
